### ISSUE TITLE
about text aligner

### ISSUE THREAD

**Issue by deyituo** [yl4579/StyleTTS2#29]
Maybe the asr module(text aligner) can be replaced with vits's idea??

**Comment by yl4579** [comment]
Yes it can, and it can be more easily adapted to other datasets, but in the original StyleTTS paper we showed that using a pretrained text aligner helps with the alignment accuracy and hence helps the performance. The alignment problem is one of the reasons VITS sometimes generates unnatural prosody (the duration is incorrect). Simply using TMA (pretrained aligner finetuned for TTS task) helps the naturalness of VITS.
